### PR TITLE
Multiple CYOAs per tutorial, linting addition

### PR DESCRIPTION
**Pull request by hexylena**
should fix it for ya @wee-snufkin, please feel free to approve/merge if you get it working with this branch :)

### CODE REVIEW

File: topics/contributing/tutorials/create-new-tutorial-content/tutorial.md [hunk 1]
```diff
@@ -1195,6 +1195,25 @@ And then they can wrap the relevant sections with a `div` block with the relevan
...
+>
+> 1. Ensure that all options are disjoint, there should not be any shared terms! (I.e if the both CYOAs need to use "STAR", please find a different way to phrase it, or even use "STAR ", it just needs to be different.)
+> 2. Provide a disambiguation term for them, passed as a parameter to all, or all but one, includes.
+>
+> This disambiguation term will affect the URL parameter, which will become `?gtn-cyoa{term}={value}`
+>
+> E.g.:
+>
+> ```
+> {% begin raw%}
```
[the remark this answers is not in the document]
Comment (wee-snufkin) [reply topics/contributing/tutorials/create-new-tutorial-content/tutorial.md:1209]: That's the problem that Gitpod had as well:
`Liquid Exception: Liquid syntax error (line 1178): Unknown tag 'begin' in topics/contributing/tutorials/create-new-tutorial-content/tutorial.md`

**Comment by wee-snufkin** [comment]
Wohoo, it works now! Thanks amazing @hexylena <3 
There's only this 'begin' tag that produces an error on Gitpod.

File: topics/contributing/tutorials/create-new-tutorial-content/tutorial.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (hexylena) [reply topics/contributing/tutorials/create-new-tutorial-content/tutorial.md]: ack, cheers. Fixed.

**Comment by wee-snufkin** [comment]
Merged into #4590

**Pull request closed without merging**